src/packet/gre.rs.in

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2016 Robert Collins <robertc@robertcollins.net>
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#[cfg(test)]
+use packet::Packet;
+
+use packet::PrimitiveValues;
+
+use pnet_macros_support::types::*;
+
+/// GRE Packet.
```

Comment:
Could you explicitly write "Generic Routing Encapsulation" somewhere please. GRE isn't particularly googleable for someone unfamiliar.